circle.yml

```diff
@@ -65,6 +65,7 @@ conda_build_test: &conda_build_test
       not xattr
       and not skeleton_pypi
       and not perl-cpan-Moo
+      and not r-cran-nmf
       and not test_expand_globs
       and not test_build_expands_wildcards
       and not numpy_build_run
```

Comment:
I think this whole `CONDABUILD_SKIP ` block should be removed.  Doesn't it get trampled by what's below in every case it's used anyway?

Reply:
Yeah, looks that way. I'll remove it.